integration-tests/tests/run-catalog-filter.js

```diff
@@ -58,5 +58,5 @@ registerTest({
   name: 'run-catalog-filter',
   fn: testCatalogFilterFn,
   timeout: 2 * 60 * 1000,
-  targetEnvironments: ['local', 'dev']
+  targetEnvironments: ['dev']
```

Comment:
This test is one of four tests that don't run against Staging backend. Other three tests seem work just fine. Jira ticket to fix all four tests to run against Staging backend is [QA-1930](https://broadworkbench.atlassian.net/browse/QA-1930)